Cargo.toml

```diff
@@ -1483,6 +1483,17 @@ description = "Illustrates how to use States to control transitioning from a Men
 category = "ECS (Entity Component System)"
 wasm = false
 
+[[example]]
+name = "derived_state"
```

Comment:
I think we should rename this to something a bit more intuitively useful.

Something like "advanced" or "complex" states would do a better job attracting the attention of the audience who should read this.

Reply:
I really liked your suggestion of `ComputedState` - so I'll go by renaming things with that, and try to make it clearer in the description as well.